### ISSUE TITLE
remark-lint-link-title-style false positive for URL containing parentheses 

### ISSUE THREAD

**Issue by pvdlg** [remarkjs/remark-lint#166]
The following markdown code wrongly returns the ``Titles should use `"` as a quote (remark-lint:link-title-style)`` error:

```md
[Select infrastructure](https://docs.travis-ci.com/user/getting-started/#Selecting-infrastructure-(optional))
```

The `(optional)` is part of the URL but it seems the rule wrongly consider it as the title.

**Comment by wooorm** [comment]
Sorry for the slow response Pierre!

Definitely a bug, would you be able to work on a fix?

**Comment by pvdlg** [comment]
I probably won't have time unfortunately. I might look at it at some point, but if anyone want to work please on it, please go ahead.

**Comment by woof1234** [comment]
guys . im being . hacked all the  tine... what securityy . company . can . i hire?

**Comment by wooorm** [comment]
@pvdlg Released!

**Comment by pvdlg** [comment]
Thanks!
